storybook/components/StoryWrapper/StoryWrapper.tsx

```diff
@@ -11,6 +11,8 @@ export interface StoryWrapperProps {
   hasNoRowTitles?: boolean
   isReversed?: boolean
   children: Array<React.ReactElement<StoryRowHeaderProps | StoryRowProps>>
+  mt?: boolean
+  mb?: boolean
```

Comment:
Bit of a nit pick, but I'd generally prefer hasMarginTop or hasMarginBottom for a boolean - I know its a little long but it just makes it clear in the props that user's won't be setting a value